L/LLVM/common.jl

```diff
@@ -15,12 +15,25 @@ const llvm_tags = Dict(
     v"12.0.1" => "980d2f60a8524c5546397db9e8bbb7d6ea56c1b7", # julia-12.0.1-4
     v"13.0.1" => "8a2ae8c8064a0544814c6fac7dd0c4a9aa29a7e6", # julia-13.0.1-3
     v"14.0.6" => "ad0184c1a2ee9508319db7c97d635a75d5336cf8", # julia-14.0.6-2
+    v"15.0.1" => "dccf0cc8852cd13d6f479afa5513db07bc586732", # julia-15.0.1-0
 )
 
 const buildscript = raw"""
 # We want to exit the program if errors occur.
 set -o errexit
 
+if [[ ("${target}" == *apple*) && ("${LLVM_MAJ_VER}" -ge "12") ]]; then
```

Comment:
Memo to self: need to fix the condition here.  I'll do it after all the build finish, so that we get a warm cache.